examples/dato/README.md

```diff
@@ -0,0 +1,59 @@
+# Makeswift Dato Integration
+
+(Link to landing page):
+https://makeswift-examples-basic-typescript.vercel.app/
+
+This guide assumes that you're familiar with Makeswift and setting up a custom host. Read here to learn more: https://docs.makeswift.com/developer/app-router/installation
+
+## Using this example
+
+To quickly try this example either [deploy to Vercel](#deploy-this-example-to-vercel) or [use our CLI](#use-this-example-locally-with-the-makeswift-cli).
+
+In addition to your Makeswift API key, add your Dato API key in either `.env.local` or your environment variables in vercel.
+
+`NEXT_PUBLIC_DATO_CMS_API_TOKEN=your_token`
```

Comment:
Can we clarify where to find this token inside of Dato? Is there any required step to generate the token or does it come by default?

Can we also move this to be listed as a requirement before we mention the two ways to deploy?

Lastly, in this section you say add to `.env.local` but we show two different ways to try the example: Vercel dpeloy button and running locally. Adding to `.env.local` is only relevant if running locally. Maybe we start by saying "You need to get an environment variable" in this section, then leave the "where do you put the environment variable" context to the two specific sections below?

Reply:
Realized you actually mentioned both. I do think it makes more sense to include the details of where to put the envirnoment variable in the appropriate section